src/util/cliffThoughtHeight.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Calculate the height that cliff thoughts need to extend.
```

Comment:
`Calculates the height of a drop target at the cliff, decreasing with depth depth. This makes it easier to drop a thought at the desired level. Returns height in ems.`